zkevm-circuits/src/bytecode_circuit/circuit.rs

```diff
@@ -1038,14 +1054,33 @@ impl<F: Field> SubCircuit<F> for BytecodeCircuit<F> {
 
     /// Return the minimum number of rows required to prove the block
     fn min_num_rows_block(block: &witness::Block) -> (usize, usize) {
-        (
-            block
-                .bytecodes
-                .values()
-                .map(|bytecode| bytecode.bytes.len() + 1)
-                .sum(),
-            block.circuits_params.max_bytecode,
-        )
+        #[cfg(feature = "dual_bytecode")]
+        {
+            // when enable feature "dual_bytecode", get two sets of bytecodes here.
+            let (first_bytecodes, second_bytecodes) = block.get_two_bytecodes();
```

Comment:
`get_two_bytecodes`.   totally misleading naming.

Reply:
i mean function naming.  not only comments.

Reply:
updated in [e15a29e](https://github.com/scroll-tech/zkevm-circuits/pull/1368/commits/e15a29e0af5909592913efb3dd1a61589baccf01)